go/enclave/crypto/crypto.go

```diff
@@ -57,16 +59,16 @@ func DecryptSecret(secret common.EncryptedSharedEnclaveSecret, privateKey *ecdsa
 	return &temp, nil
 }
 
-func EncryptSecret(pubKeyEncoded []byte, secret SharedEnclaveSecret, nodeShortID uint64) (common.EncryptedSharedEnclaveSecret, error) {
-	common.LogWithID(nodeShortID, "Encrypting secret with public key %s", gethcommon.Bytes2Hex(pubKeyEncoded))
```

Comment:
I think you can delete the `shortID` field on the node now (in go/host/node/node.go).